### ISSUE TITLE
'Test Connection' button is over-spanned

### ISSUE THREAD

**Issue by TomashKhamlai** [magento/magento2#23346]
### Preconditions (*)

1.No preconditions

### Steps to reproduce (*)

1. Navigate to Stores > Settings > Configuration > Catalog > Catalog > Catalog Search 
2. Set **Search Engine** to any **Elastic...**
3. Find **Test Connection** button and inspect it

### Expected result (*)

1. Structure button > span
```html
<button class="scalable" type="button" id="catalog_search_elasticsearch_test_connect_wizard" style="" novalidate="novalidate">
    <span id="catalog_search_elasticsearch_test_connect_wizard_result">Test Connection</span>
</button>
```

### Actual result (*)

1. Structure button > span > span > span
```html
<button class="scalable" type="button" id="catalog_search_elasticsearch_test_connect_wizard" style="" novalidate="novalidate">
    <span><span><span id="catalog_search_elasticsearch_test_connect_wizard_result">Test Connection</span></span></span>
</button>
```

**Comment by magento-engcom-team** [comment]
:white_check_mark: Confirmed by @engcom-Bravo 
 Thank you for verifying the issue. Based on the provided information internal tickets `MC-17678`, `MC-17679` were created

**Issue Available**: @engcom-Bravo, _You will be automatically unassigned. Contributors/Maintainers can claim this issue to continue. To reclaim and continue work, reassign the ticket to yourself._

**Comment by orlangur** [comment]
Thanks @TomashKhamlai! Such observations can be suggested directly as PRs.

**Comment by magento-engcom-team** [comment]
Hi @TomashKhamlai. Thank you for your report. 
The issue has been fixed in magento/magento2#23367 by @konarshankar07 in 2.3-develop branch
Related commit(s):
 - [e915796a0e75b67f9290d23352588a0d272e1629](https://github.com/magento/magento2/commit/e915796a0e75b67f9290d23352588a0d272e1629)

The fix will be available with the upcoming 2.3.3 release.
